lib/routes/account.js

```diff
@@ -1201,12 +1201,25 @@ module.exports = function (
 
         var endpointAction = 'devicesNotify'
         var stringUid = uid.toString('hex')
+
+        function catchPushError(err) {
+          // push may fail due to not found devices or a bad push action
+          // log the error but still respond with a 200.
```

Comment:
This is better than returning a 500, but would it be even more appropriate to return a 400 so that the client can know its request failed?